app/src/main/res/layout/settings_notification.xml

```diff
@@ -8,71 +8,11 @@
         android:layout_width="match_parent"
```

Comment:
I think you should remove the `ScrollView` above here, since it is already integrated in the `PreferenceFragment` and it creates strange issues with scroll bars. Also be sure to use `wrap_content` on the root layout's height, otherwise the view can be scrolled more than needed.